app/src/main/java/com/android/brewr/model/journey/Journey.kt

```diff
@@ -61,13 +64,10 @@ enum class CoffeeTaste {
 
 /** Enum class representing various coffee rates. */
 enum class CoffeeRate {
```

Comment:
Looks way simpler to have a natural number of stars rating from 1 to 5 instead of rating 1,1.5,2,2.5 ... I think this is very good. As we discussed during the standup meetings, this seems better and clearer for user